test/browser/core.js

```diff
@@ -115,7 +115,14 @@ suite('SystemJS Standard Tests', function() {
   test('Global script loading', function () {
     return System.import('fixtures/global.js').then(function (m) {
       assert.ok(m.default);
-      assert.equal(m.default.some, 'thing');
+      assert.equal(m.default.v, '2.0..0');
```

Comment:
This test changed due to the breaking nature of this change. See related https://github.com/systemjs/systemjs/blob/master/test/fixtures/browser/global.js